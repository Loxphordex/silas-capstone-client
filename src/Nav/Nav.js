import React from 'react'

export default class Nav extends React.Component {
  render() {
    return (
      <div id='qnav'>
        <h1>Quoter</h1>
      </div>
    )
  }
}