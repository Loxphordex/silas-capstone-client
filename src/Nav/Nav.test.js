import React from 'react';
import Nav from './Nav'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Enzyme from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'

Enzyme.configure({ adapter: new Adapter() })

describe('Nav', () => {
    it('renders the complete page', () => {
    const wrapper = shallow(<Nav />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})