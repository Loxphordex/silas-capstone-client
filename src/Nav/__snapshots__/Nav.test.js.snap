// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav renders the complete page 1`] = `
<div
  id="qnav"
>
  <h1>
    Quoter
  </h1>
</div>
`;
